api/serializers.py

```diff
@@ -172,6 +172,15 @@ class Meta:
 
     labels = BaseLabelSerializer(many=True, read_only=True)
 
+    created_by = UserSerializer(read_only=True)
+    created_by_id = serializers.PrimaryKeyRelatedField(
```

Comment:
I think we need to remove this field in the serializer and just put the actual user in the `created_by` field when the task is created to not allow edition on this field.

For the moment i can call the API to say the creator is an other guy because the `get_initial()` function of the view just populate field if is not already given (say me if i'm wrong)

---

PS: I think we can made this population of the field in the serializer with the `create` function. Or if is not possible with a signal on the model when create.